build.gradle

```diff
@@ -207,6 +207,9 @@ configure(projectsWithFlags('java')) {
         // Jetty ALPN support
         compileOnly libs.jetty.alpn.api
 
+        // Snappy
+        implementation 'org.xerial.snappy:snappy-java:1.1.10.3'
```

Comment:
Would you mind adding the dependency in `dependencies.toml` and using version catalog API?